packages/serverpod/lib/src/server/endpoint_dispatch.dart

```diff
@@ -52,6 +53,133 @@ abstract class EndpointDispatch {
     return connector;
   }
 
+  /// Tries to get a [MethodConnector] for a given endpoint and method name.
+  /// If the method is not found, a [MethodNotFoundException] is thrown.
+  /// If the endpoint is not found, an [EndpointNotFoundException] is thrown.
+  /// If the user is not authorized to access the endpoint, a [NotAuthorizedException] is thrown.
+  /// If the input parameters are invalid, an [InvalidParametersException] is thrown.
+  /// If the found method is not a [MethodStreamConnector], an [InvalidEndpointMethodTypeException] is thrown.
+  Future<
+      (
+        MethodStreamConnector,
+        Map<String, dynamic>,
+        List<StreamParameterDescription>
+      )> tryGetEndpointMethodStreamConnector({
+    required Session session,
+    required String endpointPath,
+    required String methodName,
+    required String body,
+    required SerializationManager serializationManager,
+    required List<String> requestedInputStreams,
+    Map<String, dynamic> additionalParameters = const {},
+  }) async {
+    var (_, method, paramMap) = await _tryGetEndpointMethodConnector(
+      session: session,
+      endpointPath: endpointPath,
+      methodName: methodName,
+      body: body,
+      serializationManager: serializationManager,
+      additionalParameters: additionalParameters,
+    );
+
+    if (method is! MethodStreamConnector) {
+      throw InvalidEndpointMethodTypeException(
+          'Method "$methodName" not found in endpoint: $endpointPath');
```

Comment:
Suggestion: This message could be pre-formatted and we just pass in the methodName and the endpointPath.

Reply:
Good idea! 👍

Reply:
[bba28a1](https://github.com/serverpod/serverpod/pull/2676/commits/bba28a1871abfb72ad2f311ec5642fef584d497c)